### PR TITLE
Make documentation a dropdown

### PR DESCRIPTION
**Pull request by dpgraham**
* Removed broken markdown links
* Dropdown item CSS

screen shot 2017-09-13 at 1 44 15 pm

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -23,7 +23,13 @@
                 <li><a href="/getting-started.html" class="localizable-link" data-localize="getting-started-nav-link">Get started</a></li>
                 <li><a href="/history.html" class="localizable-link" data-localize="history-nav-link">History</a></li>
                 <li><a href="/get-involved.html" class="localizable-link" data-localize="get-involved-nav-link">Get Involved!</a></li>
-                <li><a href="/documentation.html" class="localizable-link" data-localize="docs-nav-link">Documentation</a></li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle localizable-link" data-localize="docs-nav-link" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Documentation</a>
+                    <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                        <a class="dropdown-item" href="docs/en/about-appium/intro" href="#">English</a>
+                        <a class="dropdown-item" href="docs/cn/about-appium/intro">Chinese</a>
```
Comment (mykola-mokhnach) [inline _includes/navbar.html]: shouldn't language name itself be written in Chinese ? %)

File: scripts/repo.js [hunk 2]
```diff
@@ -177,9 +174,14 @@ async function buildDocs (pathToDocs) {
...
-    await exec('mkdocs', ['build', '--site-dir', pathToBuildDocsTo], {
-      cwd: pathToDocs,
-    });
+    try {
+      await exec('mkdocs', ['build', '--site-dir', pathToBuildDocsTo], {
+        cwd: pathToDocs,
+      });
+    } catch (e) {
+      console.log('Could not build', e.message);
+      console.log('Reason:', e.stderr);
```
Comment (mykola-mokhnach) [inline scripts/repo.js:183]: do we ignore the exception deliberately?

File: _includes/navbar.html (hunk 1 above)
Comment (jlipps) [reply _includes/navbar.html]: yes, good idea :-)

File: scripts/repo.js (hunk 2 above)
Comment (jlipps) [reply scripts/repo.js:183]: yeah, should probably still throw an error here right if we can't build?

**Review by jlipps — COMMENTED** [review]
like it. dunno if a white background looks that good though

**Comment by dpgraham** [comment]
Changes made.

**Comment by dpgraham** [comment]
Updated the CSS

screen shot 2017-09-13 at 2 01 44 pm

**Comment by mykola-mokhnach** [comment]
panda

**Pull request merged**